zerver/webhooks/slack/doc.html

```diff
@@ -0,0 +1,21 @@
+<p>Get notifications from Slack for messages on your team's public channels</p>
+<p>First, create your desired stream to receive notifications from slack</p>
+
+
+<p>
+   Next go to the following url <code>https://yourteamname.slack.com/apps/build</code>
```

Comment:
Should be:

Next, go to the following URL: <code>https://yourteamname.slack.com/apps/build</code>